### PR TITLE
add LTA011 to supported devices 

### PR DESCRIPTION
**Pull request by Elter71**
Add supports for Philips Hue white ambiance E27 1600lm with Bluetooth.

Link to product: https://www.philips-hue.com/en-gb/p/hue-white-ambiance-1-pack-e27/8719514288195

### CODE REVIEW

File: devices/philips.js [hunk 1]
```diff
@@ -591,6 +591,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp({colorTempRange: [153, 454]}),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['LTA011'],
+        model: '929002471901',
+        vendor: 'Philips',
+        description: 'Hue white ambiance E27 1600lm with Bluetooth',
+        meta: {turnsOffAtBrightness1: true},
+        extend: hueExtend.light_onoff_brightness_colortemp(),
```
Comment (Koenkk) [inline devices/philips.js:600]: Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/1004

File: devices/philips.js (hunk 1 above)
Comment (Elter71) [reply devices/philips.js]: done

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**